content/news/2022-07-29-bevy-0.8/index.md

```diff
@@ -0,0 +1,1131 @@
+3. **Maintainers can merge "completely trivial" PRs without two community approvals**. Some examples of "completely trivial": typo fixes, removing unused dependencies or code, and small "api consistency" fixes.
+4. **Controversial Decision Making on a Timer**: For all controversial PRs (including RFCs), if two maintainers approve, the PR can be labeled with a `S-Ready-For-Final-Review` label. As soon as this label is added and I have been pinged, a clock starts. If I have not responded with actionable feedback, a "snooze button" / "we aren't ready for this yet", or a veto within a month and a half (45 days), maintainers are free to merge the PR. This gives me the ability to dictate project direction in areas where that is important while also empowering maintainers to move things forward in parallel when that makes sense. We will be calibrating this approach as we go to make sure we strike the right balance between progress, quality, and consistent vision.
+5. **I still reserve the right to veto all code changes and make unilateral code changes**. This includes reverting "controversial changes" merged via (4).
+
+We've used this process for most of the last cycle and I'm loving how it is working so far: more trust, more eyes on each decision, faster development velocity. I still get to enforce consistent vision when that matters, but the community is empowered to drive efforts forward.
+
+## What is Next?
+
+* **Post Processing**: We have a lot of post processing work in the pipeline (some of it almost made it in to this release). The next release will make it easier to write post processing effects (thanks to intermediate HDR textures and a separate tonemapping step), and it will also include built in effects like "bloom" and "upscaling".
+* **Asset Preprocessing**: We will be investing heavily in our asset pipeline, with a focus on 
+* **Scene System Improvements**: This release saw a lot of investment in Reflection. We can now build then next iteration of the scene system on top of it, with a nicer scene format, nested scenes, and improved workflows.  
+* **Bevy Jam #2**: [Bevy Jam #1](https://itch.io/jam/bevy-jam-1) was a massive success: 74 entries, 1,618 ratings, and lots of good community vibes. Now that **Bevy 0.8** is released, its time to jam again! We'll release details on this soon. To stay in the loop, follow [@BevyEngine on twitter](https://twitter.com/BevyEngine) and join the [Official Bevy Discord](https://discord.gg/bevy). 
+
+## Support Bevy
+
+Sponsorships help make my full time work on Bevy sustainable. If you believe in Bevy's mission, consider sponsoring me (@cart) ... every bit helps!
```

Comment:
```suggestion
Sponsorships help make my full time work on Bevy sustainable. If you believe in Bevy's mission, consider sponsoring me (@cart). Every bit helps!
```

Reply:
I like the ellipses here. Makes it feel more conversational.